Frontend/src/pages/Dashboard.jsx

```
@@ -11,11 +13,25 @@ const Dashboard = () => {
-      <h2>Welcome, {user.email}</h2>
-      <button onClick={logout}>Logout</button>
+    <div className="h-screen w-screen flex flex-col bg-gray-100">
+      
+      {/* Top Navbar */}
+      <div className="flex justify-between items-center bg-white shadow-md px-6 py-4">
+        <div className="flex items-center space-x-2 cursor-pointer">
+          <img src={user.avatar || "/default-avatar.png"} alt="Profile" className="w-10 h-10 rounded-full" />
+          <span className="text-lg font-medium text-gray-800">Me ▼</span>
+        </div>
+        <button onClick={logout} className="bg-red-500 hover:bg-red-700 text-white font-bold py-2 px-4 rounded-lg shadow-md transition duration-300">
+          Logout
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)